js/src/setup-mc/setup-stepper/saved-setup-stepper.js

```diff
@@ -58,6 +100,15 @@ const SavedSetupStepper = ( { savedStep, onRefetchSavedStep = () => {} } ) => {
 		}
 	};
 
+	function handleFormChange( errorMessage, newValue ) {
+		this( newValue ).catch( () => createNotice( 'error', errorMessage ) );
```

Comment:
💅  it was a bit confusing to understand what `this` was doing here. I saw after  in the code that we bind  the action  to be called in `SetupFreeListings`. 

However, would be nice to add a JSDOC in this function explaining whats going on.

Reply:
Added in https://github.com/woocommerce/google-listings-and-ads/pull/1616/commits/d8ecc0824ce15ca8cbfcae5e49c0896ceb517f1b.

Reply:
Now is perfect! thanks